core/domain/user_jobs_one_off_test.py

```diff
@@ -150,6 +150,54 @@ def test_for_duplicates(self):
             user_d_contributions_model.created_exploration_ids,
             [self.EXP_ID_2])
 
+    def test_no_new_user_contributions_model_get_created_with_existing_model(
+            self):
+        model1 = exp_models.ExplorationSnapshotMetadataModel(
+            id='exp_id-1', committer_id=self.user_a_id, commit_type='create')
+
+        model1.put()
+
+        user_models.UserContributionsModel(
+            id=self.user_a_id,
+            created_exploration_ids=['exp_id']
+        ).put()
+
+        user_contributions_model = user_models.UserContributionsModel.get(
+            self.user_a_id)
+
+        self.assertEqual(
+            user_contributions_model.created_exploration_ids,
+            ['exp_id'])
+
+        self._run_one_off_job()
+
+        user_contributions_model = user_models.UserContributionsModel.get(
+            self.user_a_id)
+
+        self.assertEqual(
+            user_contributions_model.created_exploration_ids,
+            ['exp_id'])
+
+    def test_user_contributions_get_created_after_running_the_job(self):
+        model1 = exp_models.ExplorationSnapshotMetadataModel(
+            id='exp_id-1', committer_id='new_user', commit_type='create')
+
+        model1.put()
+
+        user_contributions_model = user_models.UserContributionsModel.get(
+            'new_user', strict=False)
+
+        self.assertIsNone(user_contributions_model)
+
+        self._run_one_off_job()
+
+        user_contributions_model = user_models.UserContributionsModel.get(
+            'new_user', strict=False)
+
+        self.assertEqual(
+            user_contributions_model.created_exploration_ids,
+            ['exp_id'])
+
 
 class UserDefaultDashboardOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
You can actually delete this job altogether (and its tests). We don't need it any more.

Reply:
Done